### PR TITLE
Feature/locale switcher

### PR DESCRIPTION
**Pull request by leticiawanderley**
#### What is the purpose of this pull request?
Add a locale switcher to the app store.

#### What problem is this solving?
Users could not change the store's language via its UI.

#### Screenshots or example usage
mobile

desktop

#### Types of changes
* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/resolvers/apps/utils/extraResources.ts [hunk 1]
```diff
@@ -37,32 +39,38 @@ export const getExtraResources = async (
...
   const files: string[] = getFilesToFetch(fields)
   if (files.length > 0) {
-    await Promise.map(files, filename =>
-      fetcher
-        .getAppFile(name, version, `public/metadata/${locale}/${filename}`)
-        .then(({ data }) => (extra[FILE_TO_FIELD[filename]] = data.toString()))
-        .catch(notFound(''))
+    await Promise.map(files, filename => {
+        extra[FILE_TO_FIELD[filename]] = {}
+        locales.map(locale => {
```
Comment (klzns) [inline node/resolvers/apps/utils/extraResources.ts]: Acho que ta faltando um return aqui?

File: node/resolvers/apps/utils/extraResources.ts [hunk 2]
```diff
@@ -37,32 +39,38 @@ export const getExtraResources = async (
...
+          const lang = splitLocale(locale)
+          fetcher
```
Comment (klzns) [inline node/resolvers/apps/utils/extraResources.ts]: E aqui tambem

Comment (klzns) [reply node/resolvers/apps/utils/extraResources.ts]: Nao sei, ta estranho essas promises. Acho que nao ta aguardando todas terminarem.

File: react/components/LocaleSwitcher.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      selectedLocale: findLocale(id),
+    })
+  }
+
+  handleBlur = () => {
+    this.setState({ opened: false })
+  }
+
+  handleMouseDown = e => {
+    e.preventDefault()
```
Comment (klzns) [inline react/components/LocaleSwitcher.js]: Pq isso?

Comment (leticiawanderley) [reply react/components/LocaleSwitcher.js]: Pq o dropdown de linguagens fecha no `blur` do botão de cima, se n tiver isso ele vai fechar antes do locale mudar.

**Pull request merged**